conftest.py

```diff
@@ -1,4 +1,7 @@
 import sys
+import os
 
 if sys.version_info < (3, 5):
     collect_ignore = ['tests/test_sanic_logging.py']
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'tests.django_logging.test_app.settings'
```

Comment:
Is it possible to do this in the test only, because we don't need this env variable for the other tests.